tests/integration/conftest.py

```diff
@@ -147,7 +137,7 @@ def fixture_ports() -> Ports:
     return Ports()
 
 
-@pytest.fixture(scope="module", name="zookeeper")
+@pytest.fixture(name="zookeeper")
```

Comment:
We really can't drop all module scopes fixtures, the tests are already quite slow and it's not viable to move everything to function scope.

(Don't look at the GHA test times, many tests don't run in that context.)

Thankfully we already have a solution used in core, I've added a PR to do the same in astacus: https://github.com/Aiven-Open/astacus/pull/220

Reply:
oh good, I'd far prefer to move away from `pytest-asyncio`. I'll amend this PR to not include those changes, and I'll review the versions to make sure they include only the versions included there.